docs/lang/articles/reference.md

```diff
@@ -176,22 +176,72 @@ compile time.
 
 #### Calls
 
+```
+call          ::=  primary '(' argument_list ')'
+argument_list ::=  args
+args          ::=  [arg (',' arg)*]
+arg           ::=  identifier
+```
+
+To favor simplicity, Taichi language doesn't support keyword arguments like Python.
+
 ### The power operator
 
+```
+power ::= 'ti.pow(' primary ',' primary ')' | primary ['**' u_expr]
+```
+
+Taichi predefines its own exponentiation function 'ti.pow()'. Scalars are broadcast in the case of scalar-tensor/tensor-scalar exponentiation operations, and tensor-tensor exponentiation is done elementwise without any broadcasting.
```

Comment:
It seems that scalar-tensor/tensor-scalar/tensor-tensor evaluation rules are described multiple times below. Shall we systematically describe it in a new subsection under `Basic concepts`?